### PR TITLE
Site Logo: Fix loader alignment issue

### PR DESCRIPTION
**Pull request by akasunil**
## What?

Spinner alignment was off in `Site Logo` block when image is set and we reload the screen. 
 
## Testing Instructions
- Add `Site Logo` block
- Upload image to `Site Logo` block, and Reload the page
- Note Spinner alignment.
 
## Screenshots or screencast 

|  In Trunk | In This PR |
| ------------- | ------------- |
| site-logo-spinne-before | site-logo-spinner-after
  |

### CODE REVIEW

**Comment by jasmussen** [comment]
Nice one! Seems safe to me, but if Ramon has a chance to look, would appreciate it!

File: packages/block-library/src/site-logo/edit.js [hunk 1]
```diff
@@ -654,16 +654,17 @@ export default function LogoEdit( {
 			{ controls }
 			{ mediaInspectorPanel }
 			{ ( !! logoUrl || !! temporaryURL ) && logoImage }
-			{ ! temporaryURL && ! logoUrl && ! canUserEdit && (
-				<Placeholder className="site-logo_placeholder">
+			{ ( !! isLoading ||
```
Comment (ramonjd) [inline packages/block-library/src/site-logo/edit.js]: I think `isLoading` would be a sufficient check as the value is already a boolean

File: packages/block-library/src/site-logo/edit.js [hunk 2]
```diff
@@ -654,16 +654,17 @@ export default function LogoEdit( {
...
+				( ! temporaryURL && ! logoUrl && ! canUserEdit ) ) && (
+				<Placeholder className="site-logo_placeholder" withIllustration>
 					{ !! isLoading && (
```
Comment (ramonjd) [inline packages/block-library/src/site-logo/edit.js]: Same here if you feel like changing, but optional as it's already in trunk.

**Review by ramonjd — APPROVED** [review]
Thanks for the fix!

### Before

https://github.com/user-attachments/assets/70df62fd-f44d-42a1-a84a-9fd57ac0c516




### After

https://github.com/user-attachments/assets/e35dac01-3301-4553-ad99-2c9e811c8ce5

File: packages/block-library/src/site-logo/edit.js (hunk 2 above)
Comment (akasunil) [reply packages/block-library/src/site-logo/edit.js]: I have updated the condition. Thanks @ramonjd. Ill merge the PR now.

**Pull request merged**